plugins/rtmp-services/data/services.json

```diff
@@ -1104,6 +1104,22 @@
                 "max video bitrate": 20000,
                 "max audio bitrate": 192
             }
+        },
+        {
+            "name": "Eventials",
+            "common": true,
```

Comment:
This is not a common service, this line should be omitted.

Reply:
Fixed